website/client/src/components/groups/questSidebarSection.vue

```diff
@@ -143,26 +113,43 @@
                     class="boss-health-bar"
                     :style="{width: bossHpPercent + '%'}"
                   ></div>
+                  <div
+                    class="pending-health-bar"
+                    :style="{width: pendingHpPercent + '%'}"
+                  >
+                  </div>
                 </div>
               </div>
             </div>
             <div class="row boss-details">
               <div class="col-6">
-                <span class="float-left">
-                  {{ Math.ceil(parseFloat(group.quest.progress.hp) * 100) / 100 }} / {{ parseFloat(questData.boss.hp).toFixed(2) }} <!-- eslint-disable-line max-len -->
+                <span class="float-left hp-value">
+                  <div
+                    class="svg-icon health-icon"
+                    v-html="icons.healthNoPaddingIcon"
+                  ></div>
+                  {{ Math.ceil(parseFloat(group.quest.progress.hp) * 100) / 100 }}
+                  / {{ parseFloat(questData.boss.hp) }}
+                  <strong>HP</strong>
+
                   <!-- current boss hp uses ceil so
                     you don't underestimate damage needed to end quest-->
                 </span>
               </div>
               <div
-                v-if="userIsOnQuest"
+                v-if="userIsOnQuest && user.party.quest.progress.up"
                 class="col-6"
               >
                 <!-- @TODO: Why do we not sync quest
                   progress on the group doc? Each user could have different progress.-->
```

Comment:
I have noticed this doesn't update on a click of the sync button--user needs to refresh the whole page to get updates.

Reply:
need to check how we did the other sync button refreshes again - wonder why its not working "out of the box" here too

Reply:
ok soo the `party` property is inside the fetch catch ... so I wonder what can be done to make it refresh ?

Reply:
There's a `RESYNC_COMPLETED` event that gets emitted after the fetch. Maybe update a key on the component to prompt Vue to rewrite it?

Reply:
for example `user.party....` its all from the state - which gets updated before `RESYNC_COMPLETED` 

I just don't know where / how I should reset the property  if its already from the store

Reply:
Looks like Tressley pulled this out to a GitHub ticket for us, so we can look into it later!